### ISSUE TITLE
contact email account used for admin notifications

### ISSUE THREAD

**Issue by dev-101** [osclass/Osclass#1979]
Why is currently contact email used for admin notifications - this does not make much sense.
Admin notifications should go to admin email(s), unless we add an option to specifically do it so.

**Comment by conejoninja** [comment]
Note: When there are two or mode admins/mods we should email all of them

To answer your question, probably because of legacy code, it was being done that way and we didn't change it

**Comment by dev-101** [comment]
Hi Conejo,
yes, and thankfully, that part is easy :)

**Comment by dev-101** [comment]
P.S.
I think mods should be excluded... or given the option to exclude them.

**Comment by conejoninja** [comment]
Some admin notifications (new listing, new comment,...) should be received by moderators

**Comment by dev-101** [comment]
Yes, but some e.g. new users should be received by admins only (or at least, I'd like it that way).
Also, we could add an optional 'service email' for special cases.

**Comment by dev-101** [comment]
Hi Conejo, do we need to implode array for add_bcc and list emails by e.g. comma? Or simply add them one by one, as much bcc's there are?

**Comment by conejoninja** [comment]
add_bcc should be an array, or a single address in a string 
https://github.com/osclass/Osclass/blob/master/oc-includes/osclass/utils.php#L403

**Comment by dev-101** [comment]
Thanks Conejo, I think this is it, but will need testing.
When you have some time please take a look, could be that I have forgotten something.
